src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,97 @@
-import { useState } from 'react';
+import React from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
-  // Increase the count after successful form submission
-  // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+const InitialFieldsOfMovie: Movie = {
+  title: '',
+  description: '',
+  imgUrl: '',
+  imdbUrl: '',
+  imdbId: '',
+};
+
+type Props = {
+  onAdd: (movie: Movie) => void,
+};
+
+type Field =
+'title'
+| 'description'
+| 'imgUrl'
+| 'imdbUrl'
+| 'imdbId';
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
+  const [count, setCount] = React.useState(0);
+  const [fieldsOfMovie, setFieldsOfMovie] = React
+    .useState(InitialFieldsOfMovie);
```

Comment:
Just import `useState` from `'react'`
```suggestion
  const [fieldsOfMovie, setFieldsOfMovie] = useState(InitialFieldsOfMovie);
```